pages/common/gitleaks.md

```diff
@@ -0,0 +1,32 @@
+# gitleaks
+
+> A scanner for detecting secrets and API keys leaked in Git repositories.
+> For more information, see: <https://github.com/gitleaks/gitleaks>.
```

Comment:
```suggestion
> More information: <https://github.com/gitleaks/gitleaks>.
```
This syntax is hard-coded across our pages, so I would suggest using the same here.